deploy/cr.yaml

```diff
@@ -13,6 +13,9 @@ spec:
   crVersion: 1.12.0
   image: percona/percona-server-mongodb:4.4.10-11
   imagePullPolicy: Always
+#  tls:
+#    # 90 days in hours
+#    certValidityDuration: 2160h
```

Comment:
Can I change this value?  I does not work for me.

Reply:
We don't have any logic to update certificates, we just create them. You need to set the duration before deploying the cluster or delete the certificates to make the operator create them again. Should we handle updating?

Reply:
> We don't have any logic to update certificates, we just create them. You need to set the duration before deploying the cluster or delete the certificates to make the operator create them again. Should we handle updating?

let's do the following: add current behavior  into documentation and create a separate https://jira.percona.com/browse/K8SPSMDB-655 task for future improvement.